fdbclient/BlobGranuleCommon.h

```diff
@@ -82,10 +82,11 @@ struct BlobGranuleChunkRef {
 	Optional<BlobFilePointerRef> snapshotFile; // not set if it's an incremental read
 	VectorRef<BlobFilePointerRef> deltaFiles;
 	GranuleDeltas newDeltas;
+	Optional<StringRef> tenantPrefix;
```

Comment:
Nit: since the tenant prefix is part of the key, I usually use `Optional<KeyRef>` for clarity.